spring-cloud-generator/compute-monorepo-tag.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-libraries_bom_version=$(xmllint --xpath "string(//*[local-name()='gcp-libraries-bom.version'])" ../spring-cloud-gcp-dependencies/pom.xml)
```

Comment:
Logic to parse `libraries_bom_version` is pulled out to github action instead. This is to allow the updated pom file from fork/pr branch to be fetched and used once, but having the rest of the scripts executable from main/base branch of triggered workflow.